src/main/kotlin/com/hibob/academy/dao/PetDao.kt

```diff
@@ -72,4 +73,27 @@ class PetDao @Autowired constructor(private val sql: DSLContext) {
             .fetchMap(petTable.type, DSL.count())
             .mapKeys { PetType.valueOf(it.key) }
     }
+
+    fun adoptMultiplePets(pets: List<Pet>, ownerId: Long) {
+        pets.forEach { pet ->
```

Comment:
BTW - there is another option for better performance here
to set the owner of the pet where the pet id is in the pets list (ids)

Reply:
Changed it, I hope I did it well